lib/addons/amp/index.ts

```diff
@@ -0,0 +1,131 @@
+import { KubernetesManifest } from "aws-cdk-lib/aws-eks";
+import * as aps from 'aws-cdk-lib/aws-aps';
+import { CfnWorkspaceProps } from 'aws-cdk-lib/aws-aps';
+import { ClusterAddOn, ClusterInfo } from "../../spi";
+import { dependable, loadYaml, readYamlDocument } from "../../utils";
+import { AdotCollectorAddOn } from "../adot";
+import { Construct } from 'constructs';
+
+/**
+ * Configuration options for add-on.
+ */
+ export interface AmpAddOnProps {
+    /**
+     * Name that will be used by the Addon to create the Workspace
+     * @default blueprints-amp-workspace
+     */
+    workspaceName?: string;
```

Comment:
Are any of these properties mutually exclusive? e.g. if customer passes workspaceName and remoteWriteUrl, what is our behavior? What takes precedence?

Reply:
I have added complete clarity in the documentation. This ignores any other property values passed if `prometheusRemoteWriteURL` is present.

Reply:
Can we add this to the jsdoc comments right here? I believe most developers will appreciate seeing comments from within VSCode.